app/src/main/res/values/strings.xml

```diff
@@ -429,8 +429,8 @@
     </string-array>
     <string-array name="hints_1">
         <item>One approach is to sum an infinite sequence. Another is to notice that the problem either ends or restarts after the first toss: this lets you express the answer recursively, meaning you can set up an equation where the answer appears on both sides. See https://math.stackexchange.com/questions/605083/calculate-expectation-of-a-geometric-random-variable for a solution to a similar problem.</item>
-        <item>For any event A, Pr[A] = 1 - Pr[!A] where !A means "not A", i.e. the complement of A. Sometimes it\'s easier to calculate Pr[!A], which in this case is the probability that all three birthdays are different.</item>
-        <item>Solve the previous puzzle first, and express the answer as a function of the number of people in the group. You\'ll need a calculator for this one.</item>
+        <item>For any event A, Pr[A] = 1 - Pr[!A] where !A means "not A", i.e. the complement of A. Sometimes it\'s easier to calculate Pr[!A], which in this case is the probability that all three birthdays are different. If we were asking about only two people, the answer would be 1 - 364 / 365, because the first birthday could be any day, and there are 364 equally likely days that differ from the first birthday. Do you see how to generalize that to three people? If the first two birthdays are different from one another, there are 363 ways the third birthday can differ from both of the first two. For an even more detailed hint, swipe horizontally to the following question and check its hint.</item>
```

Comment:
> the answer would be `1 - 364 / 365`, because _part1_, and _part2_.

A possible reader might relate the `1` to _part1_ and the `364/365` to _part2_, and thus get confused, especially due to the "how to generalize that to three people". One "obvious" generalization would be `1 - 364/365 - 363/365`, which would lead in a completely wrong direction.

I'd prefer the formula `1 - 365/365 * 364/365`, even though it is a bit longer. Or would that already give away too much?

Reply:
Yup, I see what you mean -- `1 - 365/365 * 364/365` is longer but it's a more generous hint.